tests/unit/s2n_key_material_test.c

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+#include "tls/s2n_prf.h"
+
+S2N_RESULT helper_generate_test_data(struct s2n_blob *test_data)
+{
+    struct s2n_stuffer test_data_stuffer = { 0 };
+    RESULT_GUARD_POSIX(s2n_stuffer_init(&test_data_stuffer, test_data));
+    for (size_t i = 0; i < S2N_MAX_KEY_BLOCK_LEN; i++) {
+        RESULT_GUARD_POSIX(s2n_stuffer_write_uint8(&test_data_stuffer, i));
+    }
+    return S2N_RESULT_OK;
```

Comment:
If you just need test data, you could use [s2n_get_public_random_data](https://github.com/aws/s2n-tls/blob/main/utils/s2n_random.h#L29)